lib/cronofy/client.rb

```diff
@@ -1339,6 +1339,34 @@ def get_smart_invite(smart_invite_id, recipient_email)
       parse_json(SmartInviteResponse, nil, response)
     end
 
+
+    # Public: Creates a link_token to allow explicitly linking of an account
+    #
+    # permissions - An Array of strings describing the permissions required for the token
+    # subs        - An Array of sub values for the account(s) the element will
+    #               be accessing
+    # origin      - The full url of the app page where the component will be used
+    #
+    # See https://docs.cronofy.com/developers/ui-elements/authentication for
+    # reference.
+    #
+    # Returns an element token
+    #
+    # Raises Cronofy::CredentialsMissingError if no credentials available.
+    # Raises Cronofy::AuthenticationFailureError if the access token is no
+    # longer valid.
+    # Raises Cronofy::TooManyRequestsError if the request exceeds the rate
+    # limits for the application.
+    def get_element_token(permissions, subs, origin)
```

Comment:
I would prefer `options = {}` style here, for one, 3 args makes me err towards that, for another passing `options` straight through leads to forwards compatibility.

Reply:
I would also consider naming as `element_token` (my preference) or perhaps `create_element_token` - `get` could be inferred as you have to create it separately.